src/JestTestRunner.ts

```diff
@@ -122,7 +122,7 @@ export default class JestTestRunner extends EventEmitter implements TestRunner {
   }
 
   private createErrorMessages(result: any[]): string[] {
-    return result.map((r: any) => r.failureMessage);
```

Comment:
To filter out any `undefined` values.

Otherwise, the `errorMessages` in the `RunResult` would be an array with as many elements as there are `TestResult` s, expect all of them that actually passed would be `undefined`.